test/workplaces.js

```diff
@@ -16,11 +16,18 @@ test('Urgency', async t => {
   const start = monthAgo.toISOString()
   const end = now.toISOString()
 
-  const urgency = await deskbookers.workplaces.urgency(17657, [
+  const what = [
```

Comment:
`what` is non-descriptive and unsightly, please choose another variable name, such as `params`